src/parser/jobs/rdm/modules/MeleeCombos.tsx

```diff
@@ -179,17 +180,20 @@ export class MeleeCombos extends Analyser {
 		}
 
 		if (action.breaksCombo) {
+			if (current) {
+				this.debug(`Action ${action.name} Breaks Combo at ${this.parser.formatEpochTimestamp(event.timestamp, 1)}`)
+			}
 			/*
 			Manafication does break combos, but the way we are currently modeling the full RDM combo isn't accurate anymore.
 			A full fix for this entails modeling mana stacks, splitting the full combo into two, and fixing the UI to display that info in a reasonable way.
 			However, as more people are starting to use Manafication after EncRedoublement (to fit multiple combos under buffs), this is a band-aid fix for now.
 			Additional Note: Event though we've now modeled the underlying combos differently, to accurately handle finishers we fabricate them appearing similar to
 			how they were when they were one large combo.  As such for now this fix is no longer a bandaid but a permanent fixture.
 			*/
-			if (action.id === this.data.actions.MANAFICATION.id &&
-				current && current.data.lastAction.action === this.data.actions.ENCHANTED_REDOUBLEMENT.id) {
-				return
-			}
+			// if (action.id === this.data.actions.MANAFICATION.id &&
+			// 	current && current.data.lastAction.action === this.data.actions.ENCHANTED_REDOUBLEMENT.id) {
+			// 	return
+			// }
```

Comment:
Did they change the behavior of Manafication to drop Red Magic stacks if you cast it between Redoublement (or Moulinet Trois) and VerFinisher?  If not, we need to keep this and make it check the AOE melee finisher too to keep the "unified combo" into the VerFinisher combo in place.

Reply:
This fix was put in place to handle for the old method of

Melee Combo --> Manafication --> Finisher.

This was because Manafication didn't clear your mana stacks, but it did give you 50|50 + the 6x potency buff stack; so if you were over 50|50 you'd spend first then manafic to rush the buff onto the finisher.  In application the melee and finisher combo are 2 separate combos, but for visualization and modeling strategies we treat them as one this is why the special handling was here.  

However, now that we don't have to worry about overcapping when using manafication; and the risk of losing out on Prefulgence if you are forced to disengage before you get to the melee end of the combo if you pop it before a finisher series, I removed it for the moment.  I still have a pending question out to the mentors on the balance, but haven't gotten a response.  As such I left it here but commented out until I do.

Reply:
But since we still have Manafication marked as breaks combo (and it **does** need to be, since it does break combos), commenting this out means that if someone does use MF in between the sword combo and the finisher combo we're now going to record that as a break.  Testing the melee combos on a dummy, it still does not break the longer combo if you use MF in between sword combo and finisher combo and the same general behavior with both the ST and AOE melee combos, so we need to not remove this and also need to add in the "don't break combo if MF used after AOE sword combo" check.

Reply:
Manafication should break combos (within the melee or within the finisher combo) but it doesn't because of a bug in the core combos module that only considers damage actions which is another issue entirely.  I'll add this back in and modify the final check on the action to be either Redoublement or Moulinet Trois.  I also am fixing the extract error I introduced in my previous check in.